### PR TITLE
Credit Netlify for OSS plan

### PR DESCRIPTION
**Pull request by davidbarsky**
Since Netlify has generously given the Tokio organization a pro plan, I'm adding the credit to the main https://tokio.rs website.

### CODE REVIEW

File: layouts/index.html [hunk 1]
```diff
@@ -132,7 +132,7 @@ <h3>
       <li><a href="/docs/overview/">About</a></li>
       <li><a href="/gsoc/">GSoC</a></li>
     </ul>
-    <p>Built with all the love in the world by <a href="https://twitter.com/carllerche" target="_blank" rel="noopener">@carllerche</a> with the help of <a href="https://github.com/tokio-rs/tokio/graphs/contributors">our contributors</a>.</p>
+    <p>Built with all the love in the world by <a href="https://twitter.com/carllerche" target="_blank" rel="noopener">@carllerche</a> with the help of <a href="https://github.com/tokio-rs/tokio/graphs/contributors">our contributors</a>. Deployed with <a href="https://netlify.com>Netlify</a>.</p>
```
Comment (LucioFranco) [inline layouts/index.html]: is this site actually deployed with netlify or is it just tracing?

Comment (davidbarsky) [reply layouts/index.html]: Just `https://tracing.rs` atm, but @carllerche mentioned that he'd like to get https://tokio.rs on Netlify as well.

**Comment by ultrasaurus** [comment]
we just hooked up Netlify this week -- still a little work left to switchover DNS, but it is already hooked up for previews (see https://github.com/tokio-rs/website/issues/373)

Would be sweet if you could rebase on master and we would be able to preview this change!

**Comment by davidbarsky** [comment]
@ultrasaurus that's a good idea! thanks for setting this site up with Netlify!

**Comment by davidbarsky** [comment]
@ultrasaurus fixed up the links and rebased! Here's the preview: https://deploy-preview-333--tokio.netlify.com/

**Comment by carllerche** [comment]
Thanks

**Pull request merged**